Package build fails for configure script without shebang

Packages can have a configure script without any shebang, in this case it should just execute it with the default shell (the safest option is probably bash). Currently it seems to error, for example the `openssl` package:

```
> rwasm::build('./openssl_2.1.1.tar.gz')
v Updated metadata database: 6.28 MB in 13 files.
v Updating metadata database ... done

> Will install 2 packages.
> Will download 3 packages with unknown size.
+ askpass   1.2.0 [dl]
+ sys       3.4.2 [dl]
v All system requirements are already installed.

i Getting 2 pkgs with unknown sizes
v Got askpass 1.2.0 (x86_64-pc-linux-gnu-ubuntu-22.04) (21.57 kB)
v Got sys 3.4.2 (x86_64-pc-linux-gnu-ubuntu-22.04) (39.83 kB)
v Downloaded 2 packages (61.40 kB) in 782ms
v Installed askpass 1.2.0  (48ms)
v Installed sys 3.4.2  (69ms)
v 2 deps: added 2, dld 2 (61.40 kB) [16.1s]
* installing *source* package 'openssl' ...
file 'configure' has the wrong MD5 checksum
** using non-staged installation
configure: ./configure.orig --host=wasm32-unknown-emscripten
emconfigure: error: './configure.orig --host=wasm32-unknown-emscripten' failed: [Errno 8] Exec format error: './configure.orig'
ERROR: configuration failed for package 'openssl'
* removing '/tmp/Rtmp8yvnLV/filec1a054c45/openssl'
```

--- a/inst/configure
+++ b/inst/configure
@@ -1,4 +1,5 @@
 #!/bin/sh
 
 # TODO: Handle non-Autoconf configure scripts that might not support --host
-emconfigure ./configure.orig --host=wasm32-unknown-emscripten
+# Bash is needed for scripts without shebang
+emconfigure bash -c './configure.orig --host=wasm32-unknown-emscripten'

